src/styles/nocturne.css

```
@@ -0,0 +1,138 @@
+/*
+ * Nocturne design-system tokens for the app, derived from the Claude Design
+ * project's _ds/nocturne styles.css with one deliberate deviation: the app
+ * uses Noto Sans (the design canvas's chosen font) instead of Inter.
+ *
+ * Every color, font, spacing, radius, and shadow in component CSS must come
+ * from these variables — never hard-code a hex or a px value a token carries.
+ * On this dark ground use ramp steps 700–900 for tinted fills, hovers, and
+ * subtle borders; 500 as a role's base; 100–300 for text on those tints.
+ */
+@import url("https://fonts.googleapis.com/css2?family=Noto+Sans:wght@400;500;600;700&display=swap");
```

Stylelint (17.14.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Noto+Sans:wght@400;500;600;700&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Noto+Sans:wght@400;500;600;700&display=swap"" (import-notation)

(import-notation)